.github/workflows/pull.yml

```diff
@@ -474,3 +474,13 @@ jobs:
       build-environment: linux-jammy-py3-clang12-executorch
       docker-image: ${{ needs.linux-jammy-py3-clang12-executorch-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-jammy-py3-clang12-executorch-build.outputs.test-matrix }}
+
+  linux-ubuntu-24_04-py3-cpu-s390x-build:
```

Comment:
In term of the workflow, this is not the appropriate place to put the build job.  Maybe we could add it to trunk if capacity allows, for example, AMD ROCm build https://github.com/pytorch/pytorch/blob/main/.github/workflows/trunk.yml#L186-L198.  Otherwise, it would need to go to a separate workflow like https://github.com/pytorch/pytorch/blob/main/.github/workflows/linux-aarch64.yml

Reply:
I'll move it to separate file then.